core/ipc/rpc/utils/src/main/java/org/opennms/core/rpc/utils/mate/FallbackScope.java

```diff
@@ -48,11 +48,23 @@ public FallbackScope(final Scope... scopes) {
 
     @Override
     public Optional<ScopeValue> get(final ContextKey contextKey) {
-        return this.scopes.stream()
+        Optional<ScopeValue> scopeValue =  this.scopes.stream()
```

Comment:
What does this change do?

Reply:
This looks like an attempt to implement recursive resolution. I would rather like to open an extra issue for this, as this need further discussion.

Reply:
This is the only way we could really overwrite alias in the requisition. 
Otherwise we couldn't use Requisition UI to overwrite alias.